contracts/Gauge.sol

```diff
@@ -50,33 +50,33 @@ contract Gauge is BaseGauge, IGauge {
     event AddedExtraReward(address reward);
     event DeletedExtraRewards(address[] rewards);
     event RemovedExtraReward(address reward);
-    event UpdatedRewardManager(address rewardManaager);
+    event UpdatedRewardManager(address rewardManager);
     event Initialized(
-        address _stakingToken,
-        address _rewardToken,
-        address _gov,
-        address _rewardManager,
-        address _ve,
-        address _veYfiRewardPool
+        address stakingToken,
+        address rewardToken,
+        address owner,
+        address rewardManager,
+        address ve,
+        address veYfiRewardPool
     );
 
     /** @notice initialize the contract
      *  @dev Initialize called after contract is cloned.
      *  @param _stakingToken The vault token to stake
      *  @param _rewardToken the reward token YFI
-     *  @param _gov governance address
+     *  @param _owner owner address
      *  @param _rewardManager reward manager address
      *  @param _ve veYFI address
      *  @param _veYfiRewardPool veYfiRewardPool address
      */
     function initialize(
         address _stakingToken,
         address _rewardToken,
-        address _gov,
+        address _owner,
         address _rewardManager,
         address _ve,
         address _veYfiRewardPool
-    ) external {
+    ) external initializer {
         assert(address(rewardToken) == address(0x0));
```

Comment:
Comment:

I think this is not required anymore since we are using the `initializer` modifier.